bbq/vm/vm.go

```diff
@@ -1508,6 +1693,7 @@ func opEmitEvent(vm *VM, ins opcode.InstructionEmitEvent) {
 }
 
 func opNewClosure(vm *VM, ins opcode.InstructionNewClosure) {
+	common.UseComputation(vm.context, common.InstructionReturnValueComputationUsage)
```

Comment:
```suggestion
	common.UseComputation(vm.context, common.InstructionNewClosureComputationUsage)
```

Reply:
Good catch! Double checked that all other instruction functions and they seem correct